platform/openide.util.ui.svg/test/unit/src/org/openide/util/svg/externalImageHref.svg

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.0//EN"
+   "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
+
+<!--
+
+   Licensed to the Apache Software Foundation (ASF) under one or more
+   contributor license agreements.  See the NOTICE file distributed with
+   this work for additional information regarding copyright ownership.
+   The ASF licenses this file to You under the Apache License, Version 2.0
+   (the "License"); you may not use this file except in compliance with
+   the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+-->
+<!-- =================================================================== -->
+<!-- A dummy document used to preload classes.                           -->
+<!-- =================================================================== -->
+
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="100" height="100" viewBox="0 0 100 100">
+    <image href="https://example.com/image.png" x="10" y="10" height="160" width="380"/>
```

Comment:
This is not valid SVG 1.0 (as which it is declared). The correct reference is:

```suggestion
    <image xlink:href="https://example.com/image.png" x="10" y="10" height="160" width="380"/>
```

The NetBeans XML Validator rightly fails the saved value. I know, that MDN says, that pure `href` is right, but that applies to SVG 2, which is is not final. And there seems to be no grammar provided for SVG2, so there is no DTD or XSD.

Reply:
Ah, I see. I added another test that has the correct xlink namespace as you say. Since these are tests of "malicious" inputs, I kept the existing non-standard one, too, as a separate test.

Reply:
Thank you